net/geth/files/geth.init

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh /etc/rc.common
+
+START=90
+STOP=10
+
+USE_PROCD=1
+PROG=/usr/bin/geth
+
+start_service() {
+    procd_open_instance
+    procd_set_param command ${PROG}
+    procd_append_param command --syncmode "light" --cache 1024
+    procd_set_param respawn
+    procd_close_instance
+}
+
+stop_service()
+{
+    service_stop ${PROG}
```

Comment:
You shouldn't need this stop_service block at all, you're not doing anything different here.  Did you add it for any reason in particular?